.github/workflows/python-app.yml

```
@@ -351,4 +351,27 @@ jobs:
           name: results_single_cell_diann
-          path: ./results_single_cell_diann
\ No newline at end of file
+          path: ./results_single_cell_diann
+
+  test_qpx:
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)